homeassistant/components/roborock/const.py

```diff
@@ -54,3 +56,10 @@
 MAP_FILENAME_SUFFIX = ".png"
 SET_VACUUM_GOTO_POSITION_SERVICE_NAME = "set_vacuum_goto_position"
 GET_VACUUM_CURRENT_POSITION_SERVICE_NAME = "get_vacuum_current_position"
+
+
+A01_UPDATE_INTERVAL = timedelta(seconds=30)
```

Comment:
From offline discussion suggestion is that 
(1)  cloud APIs should be increased to 1 minute when not cleaning. (Maybe OK to keep 30 seconds when cleaning for now)
(2) we request refresh when sending vacuum commands so that should be fine for instant updates
(3) we can fix the device resetting at 3am thing to make subscription events work more reliably in the future to reduce the need for polling